torchtune/utils/_checkpointing/_checkpointer_utils.py

```diff
@@ -53,7 +53,9 @@ def get_path(input_dir: Path, filename: str, missing_ok: bool = False) -> Path:
     return file_path
 
 
-def safe_torch_load(checkpoint_path: Path, weights_only: bool = True) -> Dict[str, Any]:
+def safe_torch_load(
+    checkpoint_path: Path, weights_only: bool = True, mmap: bool = True
```

Comment:
not related to this PR, but @ebsmothers we've come back to the API of torch.load with this change. At this point I don't think there's any use of safe_torch_load other than as a single wrapper for safetensors vs not. We probably should think about refactoring this in a clean way.